backend/index.js

```diff
@@ -0,0 +1,44 @@
+const express    = require('express');
```

Comment:
We should probably have consistent indentation throughout the build. I was using 4 spaces because my previous projects at work required it and it makes things a bit more clear imo. I'm fine going with 2 though. Just let me know.

Reply:
we can use either, my editor defaults to 2. i think bigger indentation looks gaudy, but i can get over it lol.

Reply:
Let's stick with 2 then. It's pretty standard.